swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", age=-1, condition=0):
+        super().__init__(category, age, condition)
```

Comment:
one thing we could do is hardcode "clothing" because as a Clothing child class, it's always supposed to be "clothing", right? So, let's not even give the user tha chance to mess that up!

```suggestion
    def __init__(self, age=-1, condition=0):
        super().__init__(category="Clothing", age, condition)
```

Reply:
What makes the placement of category="Clothing" different between these two examples?

Reply:
> What makes the placement of category="Clothing" different between these two examples?

well, on line 4 we've removed the parameter completely, which means users cannot define the category when creating an instance. `super` does it for them since it shouldn't ever change, and thus a user shouldn't have the power to change it anyway.

Reply:
Ah! That makes sense. This was the one part I wasn't totally sure of and just got it to a point where I knew it was working

Reply:
> Ah! That makes sense. This was the one part I wasn't totally sure of and just got it to a point where I knew it was working

Yeah, and that still works fine without it! This suggestion is just something to think about for the next time you encounter it!